### PR TITLE
Books dashboard

### PR DESCRIPTION
**Pull request by tomas-schmidt**
## Summary

- Add npm, sass, and books dashboard

## Screenshots

image

## Trello Card

https://trello.com/c/RpavBwO1/15-wbooks

### CODE REVIEW

File: wolox-books-angularjs/src/app/components/books-dashboard/books-dashboard.component.scss [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  font-style: italic;
+  font-size: 14px;
+  color: $light-grey;
+  line-height: 19px;
+  width: 240px;
+  height: 40px;
+}
+
+.search-icon {
+  height: $search-icon-size;;
```
Comment (FrankIglesias) [inline wolox-books-angularjs/src/app/components/books-dashboard/books-dashboard.component.scss]: set only one size

File: wolox-books-angularjs/src/app/components/books-dashboard/books-dashboard.component.scss [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  width: $search-icon-size;;
+}
+
+.search-icon-container {
+  background-color: $green;
+  height: $search-icon-container-size;
+  width: $search-icon-container-size;
```
Comment (FrankIglesias) [inline wolox-books-angularjs/src/app/components/books-dashboard/books-dashboard.component.scss]: same

File: wolox-books-angularjs/src/app/components/books-dashboard/books-dashboard.template.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<div class="dashboard-container">
```
Comment (FrankIglesias) [inline wolox-books-angularjs/src/app/components/books-dashboard/books-dashboard.template.html]: Use pug

File: wolox-books-angularjs/src/app/scss/application.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+@import 'equalizer.scss';
```
Comment (FrankIglesias) [inline wolox-books-angularjs/src/app/scss/application.scss]: - remove file extensions
- use wolox-equalizer.
- Check the import order

File: wolox-books-angularjs/src/app/scss/variables/_colors.scss [hunk 5]
```diff
@@ -0,0 +1,5 @@
+$lighter-grey: #f4f4f4;
+$light-grey: #9b9b9b;
+$grey: #d8d8d8;
+
```
Comment (FrankIglesias) [inline wolox-books-angularjs/src/app/scss/variables/_colors.scss]: remove this line

File: wolox-books-angularjs/src/app/scss/variables/_colors.scss [hunk 6]
```diff
@@ -0,0 +1,5 @@
+$lighter-grey: #f4f4f4;
```
Comment (FrankIglesias) [inline wolox-books-angularjs/src/app/scss/variables/_colors.scss]: use uppercase for colors

File: wolox-books-angularjs/src/app/components/books-dashboard/books-dashboard.template.html (hunk 3 above)
Comment (tomas-schmidt) [reply wolox-books-angularjs/src/app/components/books-dashboard/books-dashboard.template.html]: Pug is included in next PR

File: wolox-books-angularjs/src/app/components/navbar/navbar.component.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+angular.
+  module('navbar').
+  component('navbar', {
+    templateUrl: 'components/navbar/navbar.template.html',
+    controller: function NavbarController() {
+
```
Comment (FrankIglesias) [inline wolox-books-angularjs/src/app/components/navbar/navbar.component.js]: remove extra line

**Pull request merged**